### PR TITLE
aws-dynamodb: add page

### PR DESCRIPTION
**Pull request by Sourav-Kumar-Panda**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Sourav-Kumar-Panda** [comment]
can anyone explain what is the error here?

File: pages/common/aws-dynamodb.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- To Update an Item in the Table, you can use `update-item` command:
+
+`aws dynamodb update-item --table-name MyTable --key '{"ID": {"N": "1"}}' --update-expression "SET Name = :n" --expression-attribute-values '{":n": {"S": "Jane"}}'`
+
+- To Scan Items in the Table, you can use `scan` command:
+
+`aws dynamodb scan --table-name MyTable`
+
+- To Delete an Item from the Table, you can use `delete-item` command:
```
Comment (kant) [inline pages/common/aws-dynamodb.md:34]: To delete an item from the table,

File: pages/common/aws-dynamodb.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- To retrieve an item from a table, you use the `get-item`command:
+
+`aws dynamodb get-item \ --table-name MyTable \ --key '{"ID": {"N": "1"}}'`
+
+- To Update an Item in the Table, you can use `update-item` command:
+
+`aws dynamodb update-item --table-name MyTable --key '{"ID": {"N": "1"}}' --update-expression "SET Name = :n" --expression-attribute-values '{":n": {"S": "Jane"}}'`
+
+- To Scan Items in the Table, you can use `scan` command:
```
Comment (kant) [inline pages/common/aws-dynamodb.md:30]: To scan items in the table,

File: pages/common/aws-dynamodb.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- To add an item to a table, you use the `put-item` command:
+
+`aws dynamodb put-item --table-name TableName --item '{"AttributeName": {"S": "value"}}'`
+
+- To retrieve an item from a table, you use the `get-item`command:
+
+`aws dynamodb get-item \ --table-name MyTable \ --key '{"ID": {"N": "1"}}'`
+
+- To Update an Item in the Table, you can use `update-item` command:
```
Comment (kant) [inline pages/common/aws-dynamodb.md:26]: To update an item in the table,

**Pull request closed without merging**